File: src/main/java/com/refinedmods/refinedstorage/container/RelayContainer.java
package com.refinedmods.refinedstorage.container;

import com.refinedmods.refinedstorage.RSContainers;
import com.refinedmods.refinedstorage.blockentity.RelayBlockEntity;
import net.minecraft.world.entity.player.Player;

public class RelayContainer extends BaseContainer {
    public RelayContainer(RelayBlockEntity relay, Player player, int windowId) {
        super(RSContainers.RELAY, relay, player, windowId);

        addPlayerInventory(8, 50);
    }
}

File: src/main/kotlin/com/refinedmods/refinedstorage/container/RelayContainer.kt
package com.refinedmods.refinedstorage.container

import com.refinedmods.refinedstorage.RSContainers
import com.refinedmods.refinedstorage.api.network.grid.IGrid.slotId
import com.refinedmods.refinedstorage.tile.RelayTile
import net.minecraft.entity.player.PlayerEntity

class RelayContainer(relay: RelayTile?, player: PlayerEntity, windowId: Int) : BaseContainer(RSContainers.RELAY, relay, player, windowId) {
    init {
        addPlayerInventory(8, 50)
    }
}